tests/misc_env/install_prereq.py

```diff
@@ -173,6 +184,41 @@ def enable_rhel_rpms(ceph, distro_ver):
         )
 
 
+def enable_rhel_eus_rpms(ceph, distro_ver):
+    """
+    Setup cdn repositories for rhel systems
+    Args:
+        distro_ver: distro version details
+        ceph: ceph object
+    """
+
+    eus_repo = {"7": ("7.7", ["rhel-7-server-eus-rpms"])}
```

Comment:
There is only one value here now (RHEL 7), but can we rename this variable to `eus_repos` so it's clear that we expect to define multiple repos (RHEL 7, 8, 9, etc) here in the future?

The subscription-manager repository name will always be `rhel-7-server-eus-rpms` for all RHEL 7 EUS minor versions. RHEL 7.7 EUS, RHEL 7.9 EUS, etc will all be `rhel-7-server-eus-rpms`. We can simplify this dictionary, like
```suggestion
    eus_repos = {"7": ["rhel-7-server-eus-rpms"]}
```

Reply:
This was my initial structure, at the same i also wanted to avoid `if -else` statements like suggested below, but then I see your point, the debugging might be difficult with this one liner. I will look at the suggested approach.

Reply:
I have made the changes